<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QWidget" name="widget_5" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <item>
        <widget class="QLabel" name="label_bt">
         <property name="styleSheet">
          <string notr="true">      QLabel {
            background-color: white;       /* 背景色为白色 */
            font-size: 25px;               /* 字体大小 */
            text-align: center;            /* 文字居中对齐 */
            padding: 20px;                 /* 内边距 */
            border: 3px solid #555;        /* 边框颜色加深 */
            border-radius: 5px;            /* 边框圆角 */
        }</string>
         </property>
         <property name="text">
          <string>农业控制智能系统</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>80</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_6" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <property name="styleSheet">
             <string notr="true">#widget {
    border: 2px solid #3498db; /* 边框宽度、样式和颜色 */
    border-radius: 15px; /* 圆角弧度 */
    padding: 0px; /* 内边距 */
}
</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_wd">
                   <property name="styleSheet">
                    <string notr="true">QLabel {
    font-size: 18px; /* 字体大小 */
    color: #34495e; /* 字体颜色 */
    padding: 5px; /* 内边距 */
    qproperty-alignment: AlignCenter; /* 文本居中对齐 */
}</string>
                   </property>
                   <property name="text">
                    <string>温度</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="minimumSize">
                    <size>
                     <width>80</width>
                     <height>80</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>80</width>
                     <height>80</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">border-image: url(:/tu/感温探测器.png);</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLabel" name="label_wd_sj">
                 <property name="minimumSize">
                  <size>
                   <width>90</width>
                   <height>130</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>90</width>
                   <height>130</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <property name="styleSheet">
             <string notr="true">#widget_2{
    border: 2px solid #3498db; /* 边框宽度、样式和颜色 */
    border-radius: 15px; /* 圆角弧度 */
    padding: 0px; /* 内边距 */
}
</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_sd">
                   <property name="styleSheet">
                    <string notr="true">QLabel {
    font-size: 18px; /* 字体大小 */
    color: #34495e; /* 字体颜色 */
    padding: 5px; /* 内边距 */
    qproperty-alignment: AlignCenter; /* 文本居中对齐 */
}</string>
                   </property>
                   <property name="text">
                    <string>湿度</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="minimumSize">
                    <size>
                     <width>80</width>
                     <height>80</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>80</width>
                     <height>80</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">border-image: url(:/tu/温湿度 (1).png);</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLabel" name="label_sd_sj">
                 <property name="minimumSize">
                  <size>
                   <width>90</width>
                   <height>130</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>90</width>
                   <height>130</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_7" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <property name="styleSheet">
             <string notr="true">#widget_3{
    border: 2px solid #3498db; /* 边框宽度、样式和颜色 */
    border-radius: 15px; /* 圆角弧度 */
    padding: 0px; /* 内边距 */
}
</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_gz">
                   <property name="styleSheet">
                    <string notr="true">QLabel {
    font-size: 18px; /* 字体大小 */
    color: #34495e; /* 字体颜色 */
    padding: 5px; /* 内边距 */
    qproperty-alignment: AlignCenter; /* 文本居中对齐 */
}</string>
                   </property>
                   <property name="text">
                    <string>光照强度</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="minimumSize">
                    <size>
                     <width>80</width>
                     <height>80</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>80</width>
                     <height>80</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">border-image: url(:/tu/光照 (2).png);</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLabel" name="label_gz_sj">
                 <property name="minimumSize">
                  <size>
                   <width>90</width>
                   <height>130</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>90</width>
                   <height>130</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_4" native="true">
            <property name="styleSheet">
             <string notr="true">#widget_4{
    border: 2px solid #3498db; /* 边框宽度、样式和颜色 */
    border-radius: 15px; /* 圆角弧度 */
    padding: 0px; /* 内边距 */
}
</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_gz_2">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QLabel {
    font-size: 18px; /* 字体大小 */
    color: #34495e; /* 字体颜色 */
    padding: 5px; /* 内边距 */
    qproperty-alignment: AlignCenter; /* 文本居中对齐 */
}</string>
                   </property>
                   <property name="text">
                    <string>灯</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="minimumSize">
                    <size>
                     <width>80</width>
                     <height>80</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>80</width>
                     <height>80</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">border-image: url(:/tu/关灯.png);</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton">
                 <property name="minimumSize">
                  <size>
                   <width>90</width>
                   <height>90</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>90</width>
                   <height>90</height>
                  </size>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">        QPushButton {      
            color: white;                /* 字体颜色为白色 */		
			border-image: url(:/tu/开关按钮.png);
        }
        
        QPushButton:hover {
           
	background-color: rgb(225, 225, 225);
        }
        
        QPushButton:pressed {
            background-color: rgb(225, 225, 225);
        }</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_10" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QWidget" name="widget_8" native="true">
            <property name="styleSheet">
             <string notr="true">#widget_8
{
    border: 2px solid #3498db; /* 边框宽度、样式和颜色 */
    border-radius: 15px; /* 圆角弧度 */
    padding: 0px; /* 内边距 */
}
</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_10">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_gz_3">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QLabel {
    font-size: 18px; /* 字体大小 */
    color: #34495e; /* 字体颜色 */
    padding: 5px; /* 内边距 */
    qproperty-alignment: AlignCenter; /* 文本居中对齐 */
}</string>
                   </property>
                   <property name="text">
                    <string>警报</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="minimumSize">
                    <size>
                     <width>80</width>
                     <height>80</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>80</width>
                     <height>80</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">border-image: url(:/tu/报警.png);</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_2">
                 <property name="minimumSize">
                  <size>
                   <width>90</width>
                   <height>90</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>90</width>
                   <height>90</height>
                  </size>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">        QPushButton {      
            color: white;                /* 字体颜色为白色 */		
			border-image: url(:/tu/开关按钮.png);
        }
        
        QPushButton:hover {
           
	background-color: rgb(225, 225, 225);
        }
        
        QPushButton:pressed {
            background-color: rgb(225, 225, 225);
        }</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_9" native="true">
            <property name="styleSheet">
             <string notr="true">#widget_9{
    border: 2px solid #3498db; /* 边框宽度、样式和颜色 */
    border-radius: 15px; /* 圆角弧度 */
    padding: 0px; /* 内边距 */
}
</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_12">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_gz_4">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QLabel {
    font-size: 18px; /* 字体大小 */
    color: #34495e; /* 字体颜色 */
    padding: 5px; /* 内边距 */
    qproperty-alignment: AlignCenter; /* 文本居中对齐 */
}</string>
                   </property>
                   <property name="text">
                    <string>风扇</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_7">
                   <property name="minimumSize">
                    <size>
                     <width>80</width>
                     <height>80</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>80</width>
                     <height>80</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">border-image: url(:/tu/排风机.png);</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_3">
                 <property name="minimumSize">
                  <size>
                   <width>90</width>
                   <height>90</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>90</width>
                   <height>90</height>
                  </size>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">        QPushButton {      
            color: white;                /* 字体颜色为白色 */		
			border-image: url(:/tu/开关按钮.png);
        }
        
        QPushButton:hover {
           
	background-color: rgb(225, 225, 225);
        }
        
        QPushButton:pressed {
            background-color: rgb(225, 225, 225);
        }</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
